src/DETHRACE/common/pratcam.c

```diff
@@ -13,6 +13,9 @@
 #include "loading.h"
 #include "pd/sys.h"
 #include "sound.h"
+#ifdef DETHRACE_FIX_BUGS
```

Comment:
we don't know what the header file inclusions looked like, so i don't think we need to wrap an ifdef here